bitshuffle/bitshuffle.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+# pylint: disable=missing-docstring
```

Comment:
If it hasn’t already been done, we should open an issue to remove this and make the needed fixes. Every function *should* have a docstring.

Reply:
Pylint doesn't recognize your SHELLDOC format, I thought that would serve as a suitable docstring